app/models/edition.rb

```diff
@@ -639,7 +639,11 @@ def previously_published_documents_have_date
       errors[:base] << 'You must specify whether the document has been published before'
       @has_first_published_error = true
     elsif previously_published == 'true' # not a real field, so isn't converted to bool
-      errors.add(:first_published_at, "can't be blank") if first_published_at.blank?
+      if first_published_at.blank?
```

Comment:
I think the nested `if` is fine.  The alternative is repeating the condition, which isn't great.